rubrik_cdm/data_management.py

```diff
@@ -1424,3 +1422,58 @@ def sql_live_unmount(self, mounted_db_name, sql_instance=None, sql_host=None, fo
 
             return self.delete('v1', '/mssql/db/mount/{}?force={}'.format(mount_id, force), timeout)
 
+    def get_vsphere_live_mount(self, vm_name, timeout=15):  # pylint: ignore
+        """Get existing Live Mounts for a vSphere VM.
+
+        Arguments:
+            vm_name {str} -- The name of the mounted vSphere VM.
+
+        Keyword Arguments:
+            timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
+
+        Returns:
+            dict -- The full response of `GET /v1/vmware/vm/snapshot/mount?vm_id={vm_id}`.
+        """
+
+        self.log("get_vsphere_live_mount: Searching the Rubrik cluster for the mounted vSphere VM '{}'.".format(vm_name))
+        vm_id = self.object_id(vm_name, 'vmware', timeout=timeout)
+
+        #try:
```

Comment:
Can we remove this commented out code block?

Reply:
Comments taken out, thanks